packages/component-library/draft-templates/ComponentName.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+
+// import { Icon } from "@cultureamp/kaizen-component-library"
+// const minusIcon = require("@cultureamp/kaizen-component-library/icons/minus.icon.svg").default
```

Comment:
/nit old comments could be removed here.

Reply:
Ah just saw this – from memory are intentionally there because it's a template for new components and it's easier to uncomment those lines if you need them than go and hunt around for them in another component. Sort of like an example of how to do use imports and icons.

Pretty gross though. We could just have good docs on how to import stuff :)